alphadia/outputtransform.py

```diff
@@ -374,9 +376,44 @@ def build(
         _ = self.build_lfq_tables(folder_list, psm_df=psm_df, save=True)
         _ = self.build_library(base_spec_lib, psm_df=psm_df, save=True)
 
-        if self.config["transfer_learning"]["enabled"]:
+        if self.config["transfer_library"]["enabled"]:
             _ = self.build_transfer_library(folder_list, save=True)
 
+        if self.config["transfer_learning"]["enabled"]:
+            _ = self.build_transfer_model()
+
+    def build_transfer_model(self):
+        logger.progress("Train PeptDeep Models")
+
+        transfer_lib_path = os.path.join(
+            self.output_folder, f"{self.TRANSFER_OUTPUT}.hdf"
+        )
+        if not os.path.exists:
+            raise ValueError(
+                f"Transfer learning library was not found at {self.TRANSFER_OUTPUT}.hdf. Did you enable library generation?"
+            )
+
+        transfer_lib = SpecLibBase()
+        transfer_lib.load_hdf(
+            transfer_lib_path,
+            load_mod_seq=True,
+        )
+
+        device = "cpu"
+        if self.config["general"]["use_gpu"]:
+            device = "mps" if os.uname().sysname == "Darwin" else "gpu"
```

Comment:
here comes the `uname` bug again .. https://github.com/MannLabs/alphadia/pull/201 :-)
please create a static method that does this decision and call it here and in `libtransform.py`

Reply:
![image](https://github.com/MannLabs/alphadia/assets/6473148/8a483f59-79a6-4c8b-b543-bcc1a468e179)